<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zmqServer</class>
 <widget class="QMainWindow" name="zmqServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zmqServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblIp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;</string>
    </property>
    <property name="text">
     <string>My IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMyIp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>497</width>
       <height>10000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10000</height>
      </size>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>491</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vrtlyt">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeConClient">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <column>
     <property name="text">
      <string>Connected to server</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
